server/logger.ts

```diff
@@ -52,6 +52,22 @@ const logger = winston.createLogger({
       createSymlink: true,
       symlinkName: 'overseerr.log',
     }),
+    new winston.transports.DailyRotateFile({
+      filename: process.env.CONFIG_DIRECTORY
+        ? `${process.env.CONFIG_DIRECTORY}/logs/.machinelogs-%DATE%.json`
+        : path.join(__dirname, '../config/logs/.machinelogs-%DATE%.json'),
+      datePattern: 'YYYY-MM-DD',
+      zippedArchive: true,
+      maxSize: '20m',
+      maxFiles: '1d',
+      createSymlink: true,
```

Comment:
Also this one

Reply:
Not strictly needed; I can look for and use the json formatted log file for the day instead of one that would always be named `.machinelogs.json`. That would also have the benefit of clearing one "file" from the `config/logs` folder.